articles/nodejs-paypal-checkout/index.md

```diff
@@ -0,0 +1,187 @@
+Paypal is an online payment platform that allows users to transfer funds worldwide. Paypal makes it easier for merchants to accept online payments through their websites. Paypal provides processing for both Visa and Mastercard payments.
+
+### Project setup
+Create a directory named `paywave`. Change the working directory to the `paywave` directory created above by executing the command below.
+
+```bash
+$ mkdir paywave
+$ cd paywave
+```
+To create a Node.js application in the `paywave` directory we created above, execute the command below.
+
+```bash
+$ npm init
+```
+- `npm init` command adds the `package.json` file to our `paywave` directory, making it a Node.js application.
+
+Execute the commands below to install `express`, and `paypal-rest-sdk` into our project.
+
+```bash
+$ npm i express
+$ npm i paypal-rest-sdk
+```
+
+1. In the `paywave` directory, create a new file named `index.js`. 
+2. Add the code snippets below to the `index.js` file created above.
+   
+```javascript
+const express = require('express');
+const paypal = require('paypal-rest-sdk');
+
+paypal.configure({
+  'mode': 'sandbox', //sandbox or live
+  'client_id': '####yourclientid######',
+  'client_secret': '####yourclientsecret#####'
+});
+
+const app = express();
+
+app.get('/', (req, res) => res.sendFile(__dirname + "/index.html"));
+
+
+app.listen(PORT, () => console.log(`Server Started on ${PORT}`));
+```
+To obtain the `client_id` and `client_secret`, login into your Paypal developer account [here](https://developer.paypal.com/) and create a new app as shown in the image below.
+
+![Paypal create a new app](/pay/paypal-create-app.png)
```

Comment:
Add the correct image path i.e ![Paypal create a new app](/pay/paypal-create-app.png) should be ![Paypal create a new app](/engineering-education/nodejs-paypal-checkout/paypal-create-app.png)